modules/reserved-ips/README.md

```diff
@@ -0,0 +1,121 @@
+# IBM Reserved IP Module
+
+The module creates reserved IP's (https://cloud.ibm.com/docs/vpc?topic=vpc-managing-ip-addresses) on existing subnets. Reserved IP's can be assigned to your resources, for example VPE endpoint gateways.
+
+The module supports the following actions:
+- Create reserved IP addresses
+
+### Usage
+
+```hcl
+provider "ibm" {
+  ibmcloud_api_key = "XXXXXXXXXX" # pragma: allowlist secret
+  region           = "us-south"
+}
+
+# - Reserved IP
+module "ip" {
+  source           = "terraform-ibm-modules/vpe-gateway/ibm//modules/reserved-ips"
+  cloud_services = [
+    {
+      service_name = "kms"
+    },
+    {
+      service_name = "cloud-object-storage"
+    }
+  ]
+  subnet_zone_list = [
+    [
+      {
+        "cidr" = "10.10.10.0/24"
+        "crn" = "crn:v1:bluemix:public:is:us-south-1:a/abac0df06b644a9cabc6e44f55b3880e::subnet:0717-6ff0a6fb-e180-4048-9daf-a2f99f8740cd"
+        "id" = "0717-6ff0a6fb-e180-4048-9daf-a2f99f8740cd"
+        "name" = "vpe-vpc-instance-subnet-a"
+        "zone" = "us-south-1"
+      }
+    ],
+    [
+      {
+        "cidr" = "10.20.10.0/24"
+        "crn" = "crn:v1:bluemix:public:is:us-south-2:a/abac0df06b644a9cabc6e44f55b3880e::subnet:0727-c402f19e-ee68-41b6-90f0-a17d51f629ff"
+        "id" = "0727-c402f19e-ee68-41b6-90f0-a17d51f629ff"
+        "name" = "vpe-vpc-instance-subnet-b"
+        "zone" = "us-south-2"
+      }
+    ],
+    [
+      {
+        "cidr" = "10.30.10.0/24"
+        "crn" = "crn:v1:bluemix:public:is:us-south-3:a/abac0df06b644a9cabc6e44f55b3880e::subnet:0737-323dc004-19c5-4d27-b5bc-028b1189a316"
+        "id" = "0737-323dc004-19c5-4d27-b5bc-028b1189a316"
+        "name" = "vpe-vpc-instance-subnet-c"
+        "zone" = "us-south-3"
+      }
+    ],
+  ]
+  region           = "us-south"
+  prefix           = "vpe-default"
+  vpc_name         = "vpc-instance"
+}
+```
+
+The above will create 6 new reserved ip's as such and output them:
```

Comment:
Will that example really create 6 ips or just 3 given that 3 subnets are passed.

Reply:
3 for each service passed in `cloud_services`.